core/src/templates/layout/flex/three-column/battleship/battleship.twig

```diff
@@ -24,49 +24,49 @@
   <div class="flex-container flex-container--row-gap centered-container">
     {% block body %}
     {% if first is not empty %}
-    <div class="flex-lg-8-of-12 flex-xl-8-of-12 flex-2xl-8-of-12">
+    <div class="flex-lg-8-of-12">
       {{ first }}
     </div>
     {% endif %}
 
     {% if second is not empty %}
-    <div class="flex-md-6-of-12 flex-lg-4-of-12 flex-xl-4-of-12 flex-2xl-4-of-12">
+    <div class="flex-md-6-of-12 flex-lg-4-of-12">
```

Comment:
Haven't taken a really close look but Battleship has some issues - something to do with it's md-6 but lg-4?

Reply:
Oh, this might be because the 4-of-12 is rendered before the 6-of-12 items as the loop goes through grid first then breakpoint. I'll refactor to see if that helps.